action.yml

```diff
@@ -30,13 +30,23 @@ inputs:
     description: 'Use the GitHub API to discover current tags, which avoids the need for a git checkout, but requires `curl` and `jq`'
     required: false
     default: false
+  conventional_commits:
+    description: | 
+      Will parse the last commit message to determine the increment type
```

Comment:
Probably should be noted that it can't be used in conjunction with `calver`

Reply:
is it possible to have this as part of the scheme? so that the scheme options become: `semver`, `calver`, `conventional_commits`

Reply:
or potentially as part of the increment input? so that its options become `major`, `minor`, `patch`, `commit`

Reply:
I like the idea of it being a different scheme. 

Avoids it being in conflict `calver` for free.